policyengine_us/parameters/gov/states/al/tax/income/income_sources.yaml

```diff
@@ -0,0 +1,24 @@
+description: Alabama counts these income sources for the state adjusted gross income.
+values:
+  2021-01-01:
+    - employment_income
+    - self_employment_income
+    - interest_income
+    - dividend_income
+    - alimony_income
+    - regular_ira_distributions
+    - roth_ira_distributions
+    - pension_income
+    - rental_income
+
+metadata:
+  unit: list
+  period: year
+  label: Alabama adjusted gross income income sources
+  reference:
+    - title: 2022 Alabama Department of Revenue 
+      href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40bk.pdf#page=8
+    - title: 2021 Alabama Department of Revenue 
+      href: https://www.revenue.alabama.gov/wp-content/uploads/2022/06/21f40abk.pdf#page=7
+    - title: The Code of Alabama 1975 Section 40-18-14 
+      href: https://alisondb.legislature.state.al.us/alison/CodeOfAlabama/1975/Coatoc.htm
```

Comment:
this describes the income sources as comprising gross income, not AGI

Reply:
I believe that is the income sources for the AGI are defined as gross income as described in 40-18-14.2

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/f95d2836-085c-4588-bb37-63ac08afc3bb)

Reply:
renamed to gross_income_sources